packages/css/src/components/button/button.scss

```diff
@@ -3,47 +3,96 @@
  * Copyright (c) 2023 Gemeente Amsterdam
  */
 
-@import "../../../node_modules/@utrecht/components/button/css";
-
-@mixin reset {
-  -webkit-text-size-adjust: 100%;
-}
-
-.amsterdam-button {
+@mixin amsterdam-button {
+  border: none;
+  cursor: var(--amsterdam-action-activate-cursor);
+  display: inline-flex;
+  font-family: var(--amsterdam-button-font-family);
   font-size: var(--amsterdam-button-spacious-font-size);
+  gap: var(--amsterdam-button-gap);
   line-height: var(--amsterdam-button-spacious-line-height);
-
-  @include reset;
+  outline-offset: var(--amsterdam-button-outline-offset);
+  padding-block: var(--amsterdam-button-padding-block-start) var(--amsterdam-button-padding-block-end);
+  padding-inline: var(--amsterdam-button-padding-inline-start) var(--amsterdam-button-padding-inline-end);
+  -webkit-text-size-adjust: 100%;
 
   .amsterdam-theme--compact & {
     font-size: var(--amsterdam-breadcrumb-compact-font-size);
     line-height: var(--amsterdam-breadcrumb-compact-line-height);
   }
+
+  &:disabled,
+  &[aria-disabled="true"] {
+    cursor: var(--amsterdam-action-disabled-cursor);
+  }
+
+  &:hover:not(:disabled, [aria-disabled="true"]) {
+    cursor: var(--amsterdam-action-activate-cursor);
+  }
+}
+
+@mixin amsterdam-button-forced-color-mode {
+  @media screen and (-ms-high-contrast: active), screen and (forced-colors: active) {
+    border: 2px solid ButtonBorder; // add border because forced colors changes box-shadow to none
+  }
+}
+
+.amsterdam-button--busy {
+  cursor: var(--amsterdam-action-busy-cursor);
+}
+
+.amsterdam-button--primary {
+  background-color: var(--amsterdam-button-primary-background-color);
+  color: var(--amsterdam-button-primary-color);
```

Comment:
Should we set `box-shadow` (with the same colour as the background) for consistency and reusability with the other variants?

Reply:
Good idea